src/storage.js

```diff
@@ -0,0 +1,44 @@
+// Mimic localStorage API with chrome.storage.
+// See also: https://developer.chrome.com/docs/extensions/reference/storage/
+const storage = {
```

Comment:
Maybe using something like https://github.com/mozilla/webextension-polyfill instead could be beneficial, simply to not require maintenance of this code (although that specific might be overkill for this very small set of functions used here).

Reply:
Good to know the polyfill library, thanks. I also agree that it is overkilled. I hope to keep the core repo as minimal as possible. 😃